src/test/regress/expected/gporca_optimizer.out

```diff
@@ -12859,8 +12859,8 @@ where out.b in (select coalesce(tcorr2_d.c, 99)
    ->  Gather Motion 3:1  (slice1; segments: 3)  (cost=0.00..431.00 rows=1 width=8)
          ->  Seq Scan on tcorr1  (cost=0.00..431.00 rows=1 width=8)
    SubPlan 1  (slice0)
-     ->  Result  (cost=0.00..1324032.70 rows=1 width=8)
-           ->  Nested Loop Left Join  (cost=0.00..1324032.70 rows=1 width=8)
+     ->  Result  (cost=0.00..1324032.70 rows=3 width=8)
```

Comment:
Why 3?

Reply:
That's an actual value returned by nested loop and it has nothing in common with scaling